pmd-test/src/main/java/net/sourceforge/pmd/testframework/RuleTestRunner.java

```diff
@@ -123,23 +116,24 @@ public void evaluate() throws Throwable {
         return statement;
     }
 
-    private Statement withBefores(Statement statement) {
-        List<FrameworkMethod> befores = getTestClass().getAnnotatedMethods(Before.class);
+    private Statement withBefores(final Statement statement) {
```

Comment:
Note to maintainers: We should really make it clear where we expect `final` modifiers on local vars & parameters in our codebase. I think it's just noise in short methods, especially when it pollutes a diff like that.

Reply:
There is a PMD rule that make finals required :P It's not enabled on this project though.